Source/Core/Core/HW/EXI/EXIBrawlback.cpp

```diff
@@ -1189,137 +1191,328 @@ void CEXIBrawlback::handleStartMatch(u8* payload)
 
 void CEXIBrawlback::fixStartReplayEndianness(StartReplay& startReplay)
 {
-  swap_endian(startReplay.firstFrame);
-  swap_endian(startReplay.otherRandomSeed);
-  swap_endian(startReplay.randomSeed);
+  startReplay.firstFrame = swap_endian(startReplay.firstFrame);
```

Comment:
Can add nodiscard attribute to this swap_endian function, to make this issue that you fixed less likely to happen in the future:
https://en.cppreference.com/w/cpp/language/attributes/nodiscard